spinalcordtoolbox/reports/qc.py

```diff
@@ -801,7 +801,8 @@ def qcslice_layout(x): return x.mosaic()
     # Axial orientation, switch between the image and the centerline
     elif process in ['sct_get_centerline']:
         plane = 'Axial'
-        qcslice_type = qcslice.Axial([Image(fname_in1), Image(fname_seg)])
+        qcslice_type = qcslice.Axial([Image(fname_in1), Image(fname_seg)],
+                                     p_resample=None)
```

Comment:
On reflection, I don't think I should include this temporary fix for #3625/#2813 in this PR. Instead, I'm going to remove it for now and follow up in another PR.